Resources/Prototypes/_NF/Shipyard/orca.yml

```
@@ -0,0 +1,33 @@
+- type: vessel
+  id: Orca
+  parent: BaseVessel
+  name: NC Orca
+  description: An obselete military troop transport ship that has been retrofitted for salvage and mining operations.
+  price: 30000 #Ship is about 25500, markup was 29.3k, upped to 30k to be cleaner
+  category: Medium
+  group: Shipyard
+  shuttlePath: /Maps/_NF/Shuttles/orca.yml
+  class:
+  - Salvage
+  engine:
+  - Uranium
+
+- type: gameMap
+  id: Orca
+  mapName: 'NC Orca'
+  mapPath: /Maps/_NF/Shuttles/Orca.yml
+  minPlayers: 0
+  stations:
+    Orca:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Orca {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]

```

```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.